python/erskafka/ERSPublisher.py

```diff
@@ -0,0 +1,158 @@
+import os
+import socket
+import inspect
+import ers.issue_pb2 as ersissue
+from datetime import datetime
+from kafka import KafkaProducer
+
+from enum import Enum
+
+class SeverityLevel(Enum):
+    INFO = "INFO"
+    WARNING = "WARNING"
+    ERROR = "ERROR"
+    CRITICAL = "CRITICAL"
+    DEBUG = "DEBUG"
+
+def generate_context():
+    """Generate the context for an issue."""
+    # Walk back up the stack and find the frame for the original caller
+    frame = inspect.currentframe()
+    while hasattr(frame, "f_code"):
+        co = frame.f_code
+        filename = os.path.normcase(co.co_filename)
+        if 'ERSPublisher.py' not in filename:
+            # Found the frame of the original caller
+            break
+        frame = frame.f_back
+    
+    # If no such frame is found, default to the current frame
+    if frame is None:
+        frame = inspect.currentframe()
+
+    return ersissue.Context(
+        cwd=os.getcwd(),
+        file_name=frame.f_code.co_filename,
+        function_name=frame.f_code.co_name,
+        host_name=socket.gethostname(),
+        line_number=frame.f_lineno,
+        package_name="unknown",
+        application_name="python"
+    )
+
+def exception_to_issue(exc: Exception) -> ersissue.SimpleIssue:
+    """Converts an exception to a SimpleIssue."""
+    context = generate_context()
+    current_time = int(datetime.now().timestamp())
+    # Create the SimpleIssue with context and time only.
+    # The name and inheritance will be set in the create_issue function.
+    return ersissue.SimpleIssue(
+        context=context,
+        message=str(exc),
+        time=current_time,
```

Comment:
what unit is this? we need to be in milliseconds. 

I'm also adding the microseconds in a separate entry. We'll discuss it tomorrow